### DISCUSSION TITLE
Better getting started documentation

### DISCUSSION THREAD

**Discussion by fineberg-gliffy** [dirkwhoffmann/vAmiga#526] [General]
I think it's time for better getting started documentation?  I was able to glean some things by looking at past issues, but I am not 100% new to emulators, and still haven't gotten anything running.

Is that a possibility or did I just miss it?

**Comment by dirkwhoffmann** [comment]
What kind of documentation do you have in mind? Are you looking for a user manual, how-to pages, or code documentation?

**Comment by fineberg-gliffy** [comment]
> What kind of documentation do you have in mind? Are you looking for a user manual, how-to pages, or code documentation?

Note: I am truly grateful for your awesomely quick response, and I understand that this is likely a labor of love on stolen time...
 
I'm really looking for a no brains, never touched this before, what's the minimum I need to do step by step to get one example running.

I'm guessing something along the lines of:

1. Put the zip into your apps folder
2. Add your legal copy of the kickstart (ver) and rom (ver) to directory x, y, z
3. Do some magic to make the app start and let me configure it to find Kickstart, Rom, Ram config, and Game (or WorkBench) disk by doing some action
4. then do this to kick off running your favorite game running
5. Do this to run Game B (or configure a virtual hard disk for your paint application)

Type document - guessing 200 lines might work, and then some of us who are late to this party can start giving feedback easily...

**Comment by fineberg-gliffy** [comment]
I am also not saying that it should be dead simple instructions, because I'm not suggesting you should be spending any time at all supporting getting things up and running for general users.  Just to be clear.  I'm saying that from what I see, enabling one more tier of user (who may not be 100% familiar) might be a good idea.

**Comment by dirkwhoffmann** [comment]
I am eager to produce some kind of documentation, but I am unsure about the right form. Writing a Wiki is kind of boring. Maybe it's more fun to create one or two cool flyers. I don't know, yet.

**Comment by dirkwhoffmann** [comment]
I've added some help pages here:
https://dirkwhoffmann.github.io/vAmiga/docs/index.html

It's not complete yet, but it contains all the necessary information to get the emulator running (I think).
